doc/cabal-package.rst

```diff
@@ -1354,8 +1355,8 @@ A typical stanza for a foreign library looks like
    have any dependencies on other (Haskell) shared libraries; without
    the ``standalone`` option the generated library would have dependencies
    on the Haskell runtime library (``libHSrts``), the base library
-   (``libHSbase``), etc. Currently, ``standalone`` *must* be used on Windows
-   and *must not* be used on any other platform.
+   (``libHSbase``), etc.
+   The ``standalone`` option *must* be used on Windows (FIXME: why? is so why is this not automatic?).
```

Comment:
On Linux, ghc boot packages need to have been compiled with `-fPIC` otherwise they won't be able to be statically linked in.